openshift/_nodejs.bc.json

```diff
@@ -168,12 +168,12 @@
         },
         "resources": {
           "limits": {
-            "cpu": "250m",
-            "memory": "2Gi"
+            "cpu": "2",
+            "memory": "1Gi"
```

Comment:
This is increasing our limit by nearly 10x; do you have any concerns with our resource allocation if we have a lot of PRs building at the same time?

Reply:
Yes! We have a maximum number of 16 cores available under our tools project for builds.
As long as we don't have 8 PR's all running the builds at the same time we should be good. 

Which sometimes seems to be the case, but i'd also like for people to stop updating their branches with develop as they do and see if we can come up with some way of merging PR's one after another

Reply:
I vote we try this out for a while and if start running out of resources we can dial it down a tad bit